.github/workflows/_rust.yml

```
@@ -12,6 +12,24 @@ permissions:
   id-token: 'write'
 
 jobs:
+  bench:
+    name: bench-${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - runs-on: windows-2019 # Only platform with a benchmark right now
```

FYI: `include` is normally used to add explicit entries to the matrix. If the default N x M approach works fine too, you don't need `include`:

```suggestion
        runs-on: [
          windows-2019 # Only platform with a benchmark right now
        ]
```